### PR TITLE
Update godeps for libcontainer+cadvisor

### PR DESCRIPTION
**Pull request by derekwaynecarr**
Needed to unblock more progress on pod cgroup.

/cc @vishh @dchen1107 @timstclair

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/34403)

### CODE REVIEW

**Comment by derekwaynecarr** [comment]
@k8s-bot gci gce e2e test this

**Comment by derekwaynecarr** [comment]
@k8s-bot gci gce e2e test this

**Comment by derekwaynecarr** [comment]
@k8s-bot gci gce e2e test this

File: Godeps/Godeps.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
 	"ImportPath": "k8s.io/kubernetes",
-	"GoVersion": "go1.7",
-	"GodepVersion": "v74",
+	"GoVersion": "go1.6",
+	"GodepVersion": "v71",
```
Comment (timstclair) [inline Godeps/Godeps.json]: nit: revert these (and consider updating your go and godep versions)

**Review by timstclair — COMMENTED** [review]
lgtm

File: Godeps/Godeps.json (hunk 1 above)
Comment (derekwaynecarr) [inline Godeps/Godeps.json]: fixed.

**Comment by derekwaynecarr** [comment]
@timstclair fixed Godeps, ptal.

**Review by vishh — APPROVED** [review]

**Comment by derekwaynecarr** [comment]
@k8s-bot gci gke e2e test this

**Comment by derekwaynecarr** [comment]
@k8s-bot gke e2e test this

**Pull request merged**